hybrid/layout/pv_design_utils.py

```diff
@@ -108,7 +108,7 @@ def size_electrical_parameters(
         inverter_power=inverter_power,
         )
 
-    if n_inputs_combiner:
+    if n_inputs_combiner is not None and n_inputs_inverter is not None:
```

Comment:
This prevents a possible error that I uncovered thanks to type checking 👍

Reply:
thanks!